pkg/kv/store.go

```diff
@@ -37,6 +40,11 @@ func FormatPath(p ...string) string {
 	return strings.Join(p, string(PathDelimiter))
 }
 
+func MatchPath(path string, pathRegexp string) bool {
+	match, err := regexp.MatchString(PathBeginRegexp+pathRegexp, path)
+	return err == nil && match
+}
```

Comment:
1. No good reason. Moved to `kv_object_format`, where it is actually used, and changed to private

Reply:
2. Avoid building regex - as part of the matcer func, or all together?

Reply:
Done

Reply:
Moved back to using regex as actions package cannot be supported otherwise.
`Task` and `Run` kv entries uses the same prefix, which contains the repoID